store/sql_team_store.go

```diff
@@ -496,7 +496,13 @@ func (s SqlTeamStore) SaveMember(member *model.TeamMember) StoreChannel {
 				TeamMembers
 			WHERE
 				TeamId = :TeamId
-				AND DeleteAt = 0`, map[string]interface{}{"TeamId": member.TeamId}); err != nil {
+				AND DeleteAt = 0
```

Comment:
How often do we have users joining teams in our load tests? I believe the impacts of this should be negligible. Regardless, I should actually be able to eliminate the nested query with an inner join here